app/models/application_reference.rb

```diff
@@ -49,6 +49,18 @@ def self.not_failed
     where.not(feedback_status: %i[feedback_refused cancelled cancelled_at_end_of_cycle])
   end
 
+  def self.referees_to_chase(chase_referee_by:, rejected_chased_ids:)
+    joins(:application_form)
+      .feedback_requested
+      .where(
+        application_forms: {
+          recruitment_cycle_year: ApplicationForm.select('candidate_id').maximum(:recruitment_cycle_year),
+        },
+      )
+      .where(['requested_at < ?', chase_referee_by])
```

Comment:
fwiw, the array is superfluous here. can just have 
```
.where('requested_at < ?', chase_referee_by)
```

Reply:
I just moved the code and I didn't check 🙈. Done ♻️